src/core_plugins/region_map/public/region_map_vis_params.js

```diff
@@ -15,6 +15,14 @@ uiModules.get('kibana/region_map')
 
         $scope.collections = $scope.vis.type.editorConfig.collections;
 
+        $scope.initializeLayerSetting = function () {
+          $scope.vis.params.selectedLayer = $scope.collections.vectorLayers[0];
```

Comment:
this should be defined in the vis definition (visConfig.defaults) as we do with all the other settings, and then you don't need the ng-init part.